setup.py

```diff
@@ -45,6 +45,7 @@
 
 install_requires = [
     'invenio-base>=1.2.3',
+    'SQLAlchemy'
```

Comment:
Can you use ``invenio-db>1.0.12``? This ensures that we have the SQLAlchemy dependency only one place.